src/wordlift/mappings/pages/class-edit-mappings-page.php

```diff
@@ -206,9 +206,16 @@ private static function get_post_taxonomies_and_terms() {
 		$post_type_option = $post_type_array['post_type_option_name'];
 		// Get also the list of post types from the post_type_array.
 		$post_type_option_values = $post_type_array['post_type_option_values'];
+
+        $post_array =  self::get_post_taxonomy_key_and_value();
+        $post_option = $post_array['post_taxonomy_option_name'];
+        // Get also the list of post types from the post_type_array.
+        $post_option_values = $post_array['post_taxonomy_option_values'];
+
+
 		// Merge the post type option and post types in the taxonomy options
-		array_push( $taxonomy_options, $post_type_option );
-		$term_options = array_merge( $term_options, $post_type_option_values );
+		array_push( $taxonomy_options, $post_type_option, $post_option );
```

Comment:
In the selection box the taxonomy should not be selected, only term options should be selectable.

Refer acf , select post taxonomy and in the left side you would not be able to select the taxonomy but only the terms.
for example : category cant be selected, but the terms under this taxonomy can be selected.

you can implement a` <optgroup> `element for this.

![image (2)](https://user-images.githubusercontent.com/18109258/106108861-e1f29380-616e-11eb-8664-560b681c5a52.png)